src/explore-education-statistics-admin/src/services/PrototypeLoginService.ts

```diff
@@ -0,0 +1,25 @@
+export interface User {
+  id: string;
+  name: string;
+  permissions: string[];
+}
+
+export class PrototypeLoginService {
+  private static currentUser: User;
+
+  private static USERS: User[] = [
+    { id: 'user1', name: 'John Smith', permissions: [''] },
+    { id: 'user2', name: 'User 2', permissions: [''] },
+    { id: 'user3', name: 'User 3', permissions: [''] },
+    { id: 'user4', name: 'User 4', permissions: [''] },
+    { id: 'user5', name: 'User 5', permissions: [''] },
+  ];
+
+  public static getUserList(): User[] {
+    return PrototypeLoginService.USERS;
+  }
+
+  public static getUser(user: string) {
+    return PrototypeLoginService.USERS.filter(_ => _.id === user)[0];
```

Comment:
Avoid using `_` as a param, it's not descriptive and should avoided when you can use a better name. In this instance, `user` is ideal and naming the other `user` param `userId` is much more descriptive.

Reply:
no

Reply:
It's a self-descriptive function

Reply:
I'd use a proper name in a longer function. But in a function this small, no.

Reply:
It's pretty non-conventional to use `_` as a param unless you're intending not to use the parameter. I get that it's saving keystrokes, but I'm not interested in saving keystrokes if you can make it more explicit and consequently readable and consistent.

At the very least, I would change the function param to `userId`.

Reply:
Done